src/doppler-types.ts

```diff
@@ -13,7 +13,13 @@ export type EmptyResultWithoutExpectedErrors = { success: true } | UnexpectedErr
 
 export type PathType = 'free' | 'standard' | 'plus' | 'agencies';
 
-export type PlanType = 'free' | 'prepaid' | 'monthly-deliveries' | 'subscribers';
+export type PlanType =
+  | 'free'
+  | 'prepaid'
+  | 'monthly-deliveries'
+  | 'subscribers'
+  | 'agencies'
+  | 'demo';
```

Comment:
```suggestion
```
Demo shoud not be here, it can be mapped as free plan. I´ve removed this in my PR as well.